examples/advanced/config_search_path/conf/config.yaml

```
@@ -0,0 +1,9 @@
+defaults:
+  - dataset: imagenet
```

it makes more sense for the example to be with default to the config in conf and not additional_conf.
```suggestion
  - dataset: cifar10
```

Actually this is what your documentation page is doing. Avoid inconsistency between the example and the docs.